scripts/Sounds/MusicPlaylists.gd

```diff
@@ -0,0 +1,89 @@
+extends AudioStreamPlayer
+
+class_name MusicPlaylistsPlayer
+
+# TODO: Might make it work with saves later on
+@onready var current_playlist := String(GlobalVariables.playlists.keys()[0])
+var current_track := 0
+
+func _ready():
+	change_playlist(current_playlist)
+	play()
+
+
+func get_playlists() -> Array:
```

Comment:
```suggestion
func get_playlist_paths() -> Array:
```

Reply:
or another name because `playlist` only mean that you return an instance or it just me ^^